.github/actions/build_acapy/action.yaml

```diff
@@ -79,6 +79,6 @@ runs:
     - id: values  
       shell: bash
       run: |
-        echo "image_tag=${{ fromJSON(steps.meta.outputs.json).tags[0] }}" >> $GITHUB_OUTPUT
+        echo "image_tag=${{ inputs.ref != '' && inputs.ref ||  env.GITHUB_REF }}${{ inputs.ref }}" >> $GITHUB_OUTPUT
```

Comment:
I'm not clear on what you're expecting as the image tag.  `inputs.ref` is not required and it looks like you could include it twice.

Reply:
I took that straight from here: https://github.com/bcgov/traction/blob/main/.github/actions/build_acapy/action.yaml#L46
I just want the tag. not the full path to the image with the tag.
Originally the action outputs, for example, `ghcr.io/bcgov/traction-plugins-acapy:sha-18c0499` I just want `sha-18c0499` or whatever the image gets tagged with (the ref that gets passed to the action).